src/main/resources/templates/aeolus/bash/default.sh

```
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+export AEOLUS_INITIAL_DIRECTORY=${PWD}
+set_permissions () {
+  echo '⚙️ executing set_permissions'
+  find "${studentParentWorkingDirectoryName}" -type f -exec chmod +x "{}" +
```

Shellcheck (0.10.0)

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)